Correct EDOT Node.js docs for ELASTIC_OTEL_METRICS_DISABLED and ELASTIC_OTEL_HOST_METRICS_DISABLED (#279)
elastic/opentelemetry: docs/reference/edot-sdks/nodejs/configuration.md

| `OTEL_NODE_ENABLED_INSTRUMENTATIONS` 🔹 | [(EDOT Ref)](#otel_node_disabledenabled_instrumentations-details) Comma-separated list of instrumentations to turn on. |
| `OTEL_NODE_DISABLED_INSTRUMENTATIONS` 🔹 | [(EDOT Ref)](#otel_node_disabledenabled_instrumentations-details) Comma-separated list of instrumentations to turn off. |
| | |

> | `ELASTIC_OTEL_HOST_METRICS_DISABLED` 🔹 | [(EDOT Ref)](#elastic_otel_host_metrics_disabled-details) Disable collection of metrics done by `@opentelemetry/host-metrics` package. |

Reviewer: ```suggestion
| `ELASTIC_OTEL_HOST_METRICS_DISABLED` 🔹 | [(EDOT Ref)](#elastic_otel_host_metrics_disabled-details) Turn off collection of metrics done by `@opentelemetry/host-metrics` package. |
```

| Name | Notes |
| :--- | :---- |

> | `ELASTIC_OTEL_METRICS_DISABLED` 🔹 | [(EDOT Ref)](#deprecated-elastic_otel_metrics_disabled-details) Disable metrics export and some metrics collection by the SDK. |

Reviewer: ```suggestion
| `ELASTIC_OTEL_METRICS_DISABLED` 🔹 | [(EDOT Ref)](#deprecated-elastic_otel_metrics_disabled-details) Turn off metrics export and some metrics collection by the SDK. |
```

```bash
export OTEL_NODE_ENABLED_INSTRUMENTATIONS=http,fastify,pino,@elastic/opentelemetry-instrumentation-openai
```

> ### (Deprecated) `ELASTIC_OTEL_METRICS_DISABLED` details [deprecated-elastic_otel_metrics_disabled-details]

Reviewer: ```suggestion
### `ELASTIC_OTEL_METRICS_DISABLED` details [deprecated-elastic_otel_metrics_disabled-details] {applies_to}`section: deprecated 1.1.0`
```

> Setting `ELASTIC_OTEL_METRICS_DISABLED` to `true` will disable metrics export by the SDK and some metrics collection.  This configuration setting is deprecated (as of v1.1.0), in favor of using the following settings for finer control:

Reviewer: ```suggestion
Setting `ELASTIC_OTEL_METRICS_DISABLED` to `true` disables metrics export by the SDK and some metrics collection.  This configuration setting is deprecated as of v1.1.0 in favor of using the following settings for finer control:
```

> - To turn off the export of *any and all* metrics, set `OTEL_METRICS_EXPORTER=none`.
> - To turn off collection by the `@opentelemetry/instrumentation-runtime-node` package, set `OTEL_NODE_{DISABLED,ENABLED}_INSTRUMENTATIONS` to exclude that instrumentation. For example, `OTEL_NODE_DISABLED_INSTRUMENTATIONS=runtime-node`. [(EDOT Ref)](#otel_node_disabledenabled_instrumentations-details)
> - To turn off metrics collection by the `@opentelemetry/host-metrics` package, set `ELASTIC_OTEL_HOST_METRICS_DISABLED` to `false`.

Reviewer: ```suggestion
- To turn off the export of all metrics, set the `OTEL_METRICS_EXPORTER` environment variable to `none`.
- To turn off collection by the `@opentelemetry/instrumentation-runtime-node` package, set the `OTEL_NODE_{DISABLED,ENABLED}_INSTRUMENTATIONS` environment variable to exclude that instrumentation. For example, `OTEL_NODE_DISABLED_INSTRUMENTATIONS=runtime-node`. [(EDOT Ref)](#otel_node_disabledenabled_instrumentations-details)
- To turn off metrics collection by the `@opentelemetry/host-metrics` package, set the `ELASTIC_OTEL_HOST_METRICS_DISABLED` environment variable to `false`.
```

### `ELASTIC_OTEL_HOST_METRICS_DISABLED` details [elastic_otel_host_metrics_disabled-details]

> EDOT Node.js collects and exports [host metrics](./metrics.md#process-and-runtime-metrics) by default, using the `@opentelemetry/host-metrics` package. To disable this, set the `ELASTIC_HOST_OTEL_METRICS_DISABLED` environment variable to `true`.

Reviewer: ```suggestion
EDOT Node.js collects and exports [host metrics](./metrics.md#process-and-runtime-metrics) by default, using the `@opentelemetry/host-metrics` package. To turn off host metrics collection, set the `ELASTIC_HOST_OTEL_METRICS_DISABLED` environment variable to `true`.
```
